package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dfinity/nns",
-  "version": "0.4.0",
+  "version": "0.4.1",
```

Comment:
we actually did not release `v0.4.0` to npm.

should you revert to 0.3.0 including reverting the changelog?

Reply:
Has there been a breaking change since 0.3.0?  I assume that if the version was bumped from 0.3 to 0.4 there was a breaking change, no?

Reply:
Your are right, the changelog definitely needs to be updated. Would you prefer to revert the function name `notVotedNeurons` with version update 0.3.0 OR we leave the function name `votableNeurons` without version update?

Reply:
I good point the renaming is a breaking change. I actually like the new name.

1. Let's keep `v0.4.0` - revert the version bump in this PR

2. CHANGELOG is alright - nothing to do

3. Let's release a version next week to make things clean (I'll go through the changes to see if I need to add something to the changelog)

Works for you?

Reply:
The version commit was reverted back to 0.4.0. Thank your 👍